### ISSUE TITLE
Pipeline example not working

### ISSUE THREAD

**Issue by rhn** [geophile/marcel#17]
Example from https://www.marceltheshell.org/pipelines does not execute correctly on git version 5d63974ccd484211e6f816e3da0bcb7db7537ac8:

$ recent = [select (file: now() - file.mtime < hours(24))]
(| assign(recent, None) |) followed by excess tokens

I think pipelines are the killer feature of Marcel, so it would be nice to update the docs/fix the issue :)

**Comment by geophile** [comment]
Yes, the website is badly out of date. I would have fixed it earlier had I known marcel would hit the front page of HN! Anyway, I will start fixing the website today. 

For now, remember that (| ... |) has replaced [...] as pipeline syntax. I prefer the square brackets visually, but that syntax interferes with bash usage in glob patterns. Because I strive to for similarity with bash, the pipeline syntax had to change.
